src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/entities/CollectionEntity.kt

```diff
@@ -8,5 +8,5 @@ data class CollectionEntity(
         var title: String,
         var slug: String,
         var sort: Int,
-        var metadataFk: Int
+        var metadataFk: Int?
```

Comment:
Are we keeping this as nullable for now?

Reply:
yeahhhhhhhh unfortunately